content/src/helpers/fsWrapper.ts

```diff
@@ -0,0 +1,4 @@
+// This file was created just for testing purposes, mocking fs requires a third party lib like `mock-fs`
+// Since files like the content migrator uses both fs and fs/promises mocking it made the test code too hard to read
+// this adds no overhead and makes the test code simpler
+export * from 'fs'
```

Comment:
Did you consider passing `createReadStream` to `migrateContentFolderStructure(components, { createReadStream })`?

Reply:
That way you could pass a mocked function like this

```ts
migrateContentFolderStructure(components, { createReadStream: jest.mock() })
```

Preventing magic ✨ code injections and lifecycles

Reply:
(this is not request for a change)

Reply:
I actually thought of this file to be used by everyone using `fs` to make simpler the tests for all files, but this also an option

Reply:
It sounds like an anti-pattern and it is a code smell from my point of view. I think we should prevent adding these hacks when possible